### ISSUE TITLE
imagemodifier=-eu backwards compatibility issue

### ISSUE THREAD

**Issue by jspitz** [ypid/latex-packages#54]
With the recent update of `doclicense`, some documents cease to compile, namely those who have `imagemodifier=-eu`

For instance, this MWE

```tex
\documentclass{article}
\usepackage[T1]{fontenc}

\usepackage[%
	imagemodifier=-eu,
%	imagemodifier=-eu-88x31,% <-- Now needed
	type={CC},
	modifier={by-nc-nd},
	version={4.0}]{doclicense}

\begin{document}
\doclicenseImage
\end{document}
```

breaks with

```
! LaTeX Error: File `doclicense-CC-by-nc-nd-eu' not found.
```

This also affects some LaTeX classes which have the above setting, e.g.
http://mirrors.ctan.org/macros/latex/contrib/univie-ling/tex/univie-ling-wlg.cls

I think `doclicense` should make `imagemodifier=-eu` link to `imagemodifier=-eu-88x31`.

Probably it would have been better to introduce a new option for the file size selection.

**Comment by ypid** [comment]
Thanks very much for reporting! There is a reason why this has slipped my testing: I have TeX Live installed which includes an older version of the package. So when I changed the image filenames, the older filenames were still present for `kpsewhich` to find.

That is a bad development environment for this package because it makes use of `kpsewhich` more than other packages. I fixed it by moving the system wide doclicense package away so that `kpsewhich` only finds the development version. This is done on start of my development VM on Qubes OS, so issues like this should not occur in the future.

The documentation was also wrong. I propose #55 to address this bug. Does that look good to you? If yes, I will release this in the next few days.

> imagemodifier=-eu

The package should have sane defaults and you should be able to let the package figure out `imagemodifier`. Does this not work for your class?

> Probably it would have been better to introduce a new option for the file size selection.

I considered that but I went with reusing the `imagemodifier` to have flexibility for other license types. One issue with `imagemodifier` is that it cannot really be extended with another dimension because that would break backwards compatibility for people specifying `imagemodifier`. I will keep this design flaw in mind and rework as needed.

**Comment by jspitz** [comment]
> I propose #55 to address this bug. Does that look good to you? If yes, I will release this in the next few days.

Looks good (though I have not tested).

> > imagemodifier=-eu
> 
> The package should have sane defaults and you should be able to let the package figure out `imagemodifier`. Does this not work for your class?

I seem to remember it did not when I set up these options.

Thanks for the swift reaction (the class mentioned above has also been updated meanwhile).

**Comment by ypid** [comment]
> Looks good (though I have not tested).

Thanks. Should be good to release then.

> Thanks for the swift reaction (the class mentioned above has also been updated meanwhile).

Seen that, but I guess this fix should go out as fast as possible.

I just released it as v2.0.1 and uploaded the release to CTAN.

**Comment by jspitz** [comment]
Thanks!
